File: 1-Iniciante/javascript/1052_mes.js
var entry = parseInt(require("fs").readFileSync("/dev/stdin", "utf8"));
var meses = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']
console.log(meses[entry-1])

File: 1-Iniciante/c/1052_mes.c
#include <stdio.h>

int main() {
    char meses[12][11] = {"January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"};
    int informado;
    scanf("%d", &informado);
    printf("%s\n", meses[informado-1]);
    
    return 0;
}